### PR TITLE
[serve] Support batches for ImportedBackends

### LINKED ISSUE #13343: [serve] ImportedBackend cannot accept batches
### What is the problem?

*Ray version and other system information (Python version, TensorFlow version, OS):*

### Reproduction (REQUIRED)
Please provide a short code snippet (less than 50 lines if possible) that can be copy-pasted to reproduce the issue. The snippet should have **no external library dependencies** (i.e., use fake or mock data / environments):

If the code snippet cannot be run by itself, the issue will be closed with "needs-repro-script".

- [ ] I have verified my script runs in a clean environment and reproduces the issue.
- [ ] I have verified the issue also occurs with the [latest wheels](https://docs.ray.io/en/master/installation.html).

### PR DESCRIPTION
**Pull request by edoakes**
## Related issue number

Closes https://github.com/ray-project/ray/issues/13343

## Checks

- [ ] I've run `scripts/format.sh` to lint the changes in this PR.
- [ ] I've included any doc changes needed for https://docs.ray.io/en/master/.
- [ ] I've made sure the tests are passing. Note that there might be a few flaky tests, see the recent failures at https://flakey-tests.ray.io/
- Testing Strategy
   - [ ] Unit tests
   - [ ] Release tests
   - [ ] This PR is not tested :(

### CODE REVIEW

**Comment by simon-mo** [comment]
This seems like we are forcing all ImportedBackend to accept batches only?

**Comment by edoakes** [comment]
@simon-mo it doesn't force them all to accept batches, it just marks them all with the label so there is no validation for that option.

File: python/ray/serve/backends.py [hunk 1]
```diff
@@ -26,6 +27,10 @@ def reconfigure(self, *args, **kwargs):
                 # proxy it manually.
                 return self.wrapped.reconfigure(*args, **kwargs)
 
+            @serve.accept_batch
```
Comment (simon-mo) [inline python/ray/serve/backends.py:30]: add a comment saying this just bypass a check and it should works for non-batch method as well?

**Review by simon-mo — APPROVED** [review]

File: python/ray/serve/backends.py (hunk 1 above)
Comment (edoakes) [reply python/ray/serve/backends.py:30]: will do!

**Pull request merged**